### DISCUSSION TITLE
What is the best practice to query a user by email?

### DISCUSSION THREAD

**Discussion by kierangilliam** [strapi/strapi#6151] [Legacy Topics]
I am trying to query a user by email, but the issue is that the filters seem to be case sensitive. What is the best way to query a user by email?

```
const member = await strapi.query('user', 'users-permissions').find({ email })
```

**Comment by kierangilliam** [comment]
Still encountering this issue when trying to perform case insensitive queries, any ideas?

**Comment by altaygileren** [comment]
Have you tried just using .toLowerCase() on the filter?

**Comment by lauriejim** [comment]
In the register function or Strapi we lowercase emails `params.email = params.email.toLowerCase();`
So in your search, your will have to do the same to make sure your filter match.

`const member = await strapi.query('user', 'users-permissions').find({ email: email.toLowerCase() })`
